src/main/studio/kdb/Config.java

```diff
@@ -1246,26 +1246,28 @@ private Font get(String key, Font defaultValue) {
         return new Font(name, style, size);
     }
 
-    public Font getFont(String key) {
-        return get(key, (Font) checkAndGetDefaultValue(key, ConfigType.FONT));
+    public String getFontName() {
+        return p.getProperty("font.name", "Monospaced");
     }
 
-    // Returns whether the value was changed
-    public boolean setFont(String key, Font value) {
-        Font currentValue = getFont(key);
-        if (currentValue.equals(value))
-        if (currentValue == value) {
-            return false;
-        }
-        p.setProperty(key + ".name", value.getName());
-        setInt(key + ".size", value.getSize());
+    public int getFontSize() {
+        return Integer.parseInt(p.getProperty("font.size","14"));
+    }
 
-        int style = value.getStyle();
-        if (style < 0 || style > 3) style = 0; // Not sure if it is posible
-        FontStyle fontStyle = FontStyle.values()[style];
-        setEnum(key + ".style", fontStyle);
+    public Font getFont() {
+        String name = getFontName();
+        int  size = getFontSize();
 
+        Font f = new Font(name, Font.PLAIN, size);
+        setFont(f);
+
+        return f;
+    }
+
+    public void setFont(Font f) {
+        p.setProperty("font.name", f.getFamily());
+        p.setProperty("font.size", "" + f.getSize());
         save();
-        return true;
     }
+
```

Comment:
I wanted to simplify definition of properties in the Config (however i didn't finish with all the config)... This (old) approach would result that we need to copy/paste the code for any future getResultPaneFont(), getAnyOtherFont(), etc.

My idea is to implement getXXX and setXXX once for a type and then adding a new property would one line.

I started to do this for Font.. However now I see that it wasn't finished... You can complete yourself. Or I can raise a PR later if you want...

Here is how to complete... Restore my version of getFont and setFont. And you can use CONFIG.getFont(Config.FONT_EDITOR) and CONFIG.setFong(Config.FONT_EDITOR, font). But in addition you need to fix:
```
Config.configDefault:
else if (type == ConfigType.FONT) {
            Font font = (Font) defaultValue;
            configDefault(key + ".size", ConfigType.INT, font.getSize());
            configDefault(key + ".name", ConfigType.STRING, font.getName());
            configDefault(key + ".style", ConfigType.ENUM, FontStyle.getStyle(font.getStyle()));
        }else if (type == ConfigType.FONT) {
            Font font = (Font) defaultValue;
            configDefault(key + ".size", ConfigType.INT, font.getSize());
            configDefault(key + ".name", ConfigType.STRING, font.getName());
            configDefault(key + ".style", ConfigType.ENUM, FontStyle.getStyle(font.getStyle()));
        }

Config.FontStyle:
        public static FontStyle getStyle(int fontStyle) {
            return FontStyle.values()[fontStyle];
        }